defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+gocd_common_user: go
+gocd_common_userid: 5000
+gocd_common_group: go
+gocd_common_groupid: 5000
+gocd_common_debian_repository: http://dl.bintray.com/gocd/gocd-deb/
```

Comment:
Please don't use bintray, GoCD has moved away from that. 
Ref - https://groups.google.com/d/msg/go-cd/nadSZDN4eM0/2OwJwvygBwAJ